README.md

```diff
@@ -14,3 +14,6 @@ You will need at least a *free* account on [Hazelcast Cloud](https://cloud.hazel
         
 - <h3><a href="https://github.com/hazelcast/hazelcast-cloud-code-samples/tree/master/serverless-fraud-detection">serverless-fraud-detection</a></h3>
         Serverless fraud detection, using Node.js and Amazon Lambda        
+
+  <h3><a href="https://github.com/hazelcast/hazelcast-cloud-code-samples/tree/master/spring-sample">spring-sample</a></h3>
```

Comment:
We should make sure to use Hazelcast Viridian instead of Hazelcast Cloud in all READMEs.

Reply:
only in text messages or even in application properties?

Reply:
Will it work if we rename those?

Reply:
done